CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to the vscode-dbt-power-user extension! We greatly appreciate the dedication of our community members in helping us enhance this project.
```

Comment:
use dbt Power User as the name of the extension.

Reply:
updated all references